### ISSUE TITLE
Lock screen mode quality/resolution is really bad

### ISSUE THREAD

**Issue by tapir** [GreatApo/GreatFit#66]
Using the latest apk on a Verge
Compared to any other watchface I've tried, the lockscreen on greatfit is by far the lowest quality one. The "Better Resolution" setting doesn't seem to have any effect. Is there a way to bump it? 

Thanks

**Comment by GreatApo** [comment]
The watchface is scaled up from pace/stratos resolution thats why it is worse than normal watchfaces. The "better resolution" has to do with the intermediate screen when you raise your hand on pace/stratos.
No improvements can be done without messing with the code, but yes, there is space for improvements.

**Comment by tapir** [comment]
The watchface itself is quite good actually (resolution wise) I'm talking about the lock screen which I guess what you call the intermediate screen? The one you see when the screen turns on when you lift your arm.

**Comment by tapir** [comment]
Upon further investigation I need to clarify that I didn't mean the lock screen. What I meant from the start was 26w (wrist up screen)

**Comment by GreatApo** [comment]
Yeah, that's what I ment with the intermediate screen. Yeah, it's quite bad. I will say it is probably related to the libs I am using. Because those libs are compiled, it's hard to make everything work and even harder to fix these bugs :/
Generally, verge doesn't need an 8c and 26w screen. These are just there because the system (and probably the drivers) are a port from pace and Stratos. Theoretically, the color and function limitations should be there.

**Comment by tapir** [comment]
The original watchfaces that came with the watch don't have this issue though. It might be a pointer on how to solve it.
Thanks!
